### PR TITLE
KO: update ko/beta/tutorials/keras/feature_columns.ipynb

### PR DESCRIPTION
**Pull request by rickiepark**
Sync with en file.
Thanks.

### CODE REVIEW

File: site/ko/beta/tutorials/keras/feature_columns.ipynb [hunk 1]
```diff
@@ -701,7 +701,8 @@
         "\n",
         "model.compile(optimizer='adam',\n",
         "              loss='binary_crossentropy',\n",
-        "              metrics=['accuracy'])\n",
+        "              metrics=['accuracy'],\n",
+        "              run_eagerly=True)\n",
```
Comment (lamberta) [inline site/ko/beta/tutorials/keras/feature_columns.ipynb]: @yashk2810 Why does this need to be set in TF2? What does it mean if it's eager by default?

**Comment by lamberta** [comment]
Thanks, Haesun, for the update.
Now trying to understand the reason we need `run_eagerly=True` here

File: site/ko/beta/tutorials/keras/feature_columns.ipynb (hunk 1 above)
Comment (yashk2810) [reply site/ko/beta/tutorials/keras/feature_columns.ipynb]: compile and fit runs in function mode by default for better performance out of the box. 

Eager by default means that outside fit and compile you can still debug and print stuff in eager mode.

Comment (lamberta) [reply site/ko/beta/tutorials/keras/feature_columns.ipynb]: Thanks, Yash.
I get the performance benefits, but the notebook breaks without this—is that WAI?

Comment (yashk2810) [reply site/ko/beta/tutorials/keras/feature_columns.ipynb]: Yup, I have a bug filed against that and it was suggested to use run_eagerly there.

Comment (lamberta) [reply site/ko/beta/tutorials/keras/feature_columns.ipynb]: This has been fixed in `tf-nightly` and en/ notebook updated in https://github.com/tensorflow/docs/commit/ac65fe769fedf44a3afe4592f56182831cc72913

@rickiepark Can you update this notebook to use `tf-nightly-2.0-preview` and remove `run_eagerly=True`. Thanks!

Comment (rickiepark) [reply site/ko/beta/tutorials/keras/feature_columns.ipynb]: @lamberta I just pushed a commit for it. Thanks!

**Pull request merged**